cmd/diff.go

```diff
@@ -116,6 +130,20 @@ func diffImages(image1Arg, image2Arg string, diffArgs []string) error {
 	return nil
 }
 
+func diffFile(imageMap map[string]*pkgutil.Image, image1Arg, image2Arg string) error {
+
+	image1FilePath := imageMap[image1Arg].FSPath + filename
```

Comment:
these should probably use something like `path.Join` https://golang.org/pkg/path/#Join